java-surefire/src/main/java/org/sonar/plugins/surefire/SurefireExtensions.java

```diff
@@ -34,9 +34,23 @@ private SurefireExtensions() {
 
   public static List getExtensions() {
     return ImmutableList.of(
+        /**
+         * @deprecated
+         * @since 4.11
```

Comment:
why `@since` here? It was not introduced in 4.11. Do you mean to say that it is deprecated since 4.11? I would just add a free text after `@deprecated` tag